.github/workflows/kani.yml

```diff
@@ -96,39 +96,14 @@ jobs:
         env:
           RUST_TEST_THREADS: 1
 
-  bookrunner:
+  documentation:
     runs-on: ubuntu-20.04
     permissions:
       contents: write
     steps:
       - name: Checkout Kani
         uses: actions/checkout@v4
 
-      - name: Setup Kani Dependencies
-        uses: ./.github/actions/setup
-        with:
-          os: ubuntu-20.04
-
-      - name: Build Kani
-        run: cargo build-dev
-
-      - name: Install book runner dependencies
-        run: ./scripts/setup/install_bookrunner_deps.sh
-
-      - name: Generate book runner report
-        run: cargo run -p bookrunner
-        env:
-          DOC_RUST_LANG_ORG_CHANNEL: nightly
-
-      - name: Print book runner text results
-        run: cat build/output/latest/html/bookrunner.txt
-
-      - name: Print book runner failures grouped by stage
-        run: python3 scripts/ci/bookrunner_failures_by_stage.py build/output/latest/html/index.html
-
-      - name: Detect unexpected book runner failures
-        run: ./scripts/ci/detect_bookrunner_failures.sh build/output/latest/html/bookrunner.txt
-
       - name: Install book dependencies
         run: ./scripts/setup/ubuntu/install_doc_deps.sh
```

Comment:
You can remove the commented out `cargo install ...` in `install_doc_deps.sh`

Reply:
No, that's required to render the graph in `docs/src/benchcomp-cli.md`, as @karkhaz reminded me. But then he told me that we should be able to remove any litani-related left-overs, so I'll be doing this in an extra commit in this PR.

Reply:
For some reason, the graph referred to in https://model-checking.github.io/kani/benchcomp-cli.html doesn't appear rendered for me. Is anyone seeing the same?

Reply:
Yes, that's because of a separate PR that removed the mdbook-graphviz crate. See [this](https://github.com/model-checking/kani/issues/3005)